server/src/main/java/org/opensearch/action/admin/indices/upgrade/post/TransportUpgradeSettingsAction.java

```diff
@@ -102,14 +102,14 @@ protected AcknowledgedResponse read(StreamInput in) throws IOException {
     }
 
     @Override
-    protected void masterOperation(
+    protected void clusterManagerOperation(
         final UpgradeSettingsRequest request,
         final ClusterState state,
         final ActionListener<AcknowledgedResponse> listener
     ) {
         UpgradeSettingsClusterStateUpdateRequest clusterStateUpdateRequest = new UpgradeSettingsClusterStateUpdateRequest().ackTimeout(
             request.timeout()
-        ).versions(request.versions()).masterNodeTimeout(request.masterNodeTimeout());
+        ).versions(request.versions()).masterNodeTimeout(request.clusterManagerNodeTimeout());
```

Comment:
Same here, seems like `UpgradeSettingsClusterStateUpdateRequest` needs a rename of `masterNodeTimeout`?

Reply:
Thank for reviewing my PR! 
Yes, all the public methods which don't belong to the package or class name with `master` word will be changed in issue #3544